nautobot/extras/homepage.py

```diff
@@ -5,11 +5,7 @@
 
 def get_job_results(request):
     """Callback function to collect job history for panel."""
-    return (
-        JobResult.objects.filter(status__in=JobResultStatusChoices.READY_STATES)
-        .defer("data")
-        .order_by("-date_done")[:10]
-    )
+    return JobResult.objects.filter(status__in=JobResultStatusChoices.READY_STATES).order_by("-date_done")[:10]
```

Comment:
Should we `.defer("result")` now instead of removing the defer? See #3788 for similar issue.